src/Resources/views/Collector/mercure.html.twig

```diff
@@ -0,0 +1,83 @@
+{% extends '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% import _self as helper %}
+
+{% block toolbar %}
+    {% if collector.count > 0 %}
+        {% set icon %}
+            {{ include('@Mercure/Icon/mercure.svg') }}
+            <span class="sf-toolbar-value">{{ collector.count }}</span>
+        {% endset %}
+
+        {{ include('@WebProfiler/Profiler/toolbar_item.html.twig', { link: 'mercure' }) }}
+    {% endif %}
+{% endblock %}
+
+{% block menu %}
+    <span class="label{{ collector.count == 0 ? ' disabled' }}">
+        <span class="icon">{{ include('@Mercure/Icon/mercure.svg') }}</span>
+        <strong>Mercure</strong>
+    </span>
+{% endblock %}
+
+{% block panel %}
+    {% import _self as helper %}
+
+    <h2>Messages</h2>
+
+    {% if collector.count == 0 %}
+        <div class="empty">
+            <p>No messages have been collected.</p>
+        </div>
+    {% else %}
+        <div class="sf-tabs">
+            {% for hub, publisher in collector.publishers %}
+                <div class="tab">
+                    <h3 class="tab-title">{{ hub }}<span class="badge">{{ publisher.count }}</span></h3>
+                    <div class="tab-content">
+                        <p class="text-muted">{{ hub }}</p>
```

Comment:
It's useless to display the hub name here, as it's already displayed on the tab name (line 36)